app/assets/javascripts/admin/application.js

```diff
@@ -16,6 +16,16 @@
 
 $(document).on('turbolinks:load', function() {
 
+  if ($('#age_report').length) {
```

Comment:
We already have a admin/gobierto_budget_consultations/application.js - lets put this there.